packages/i18n/src/locales/en/options/bee.yml

```diff
@@ -72,4 +72,28 @@ neckTieColours:
 
 crossBackTies:
   title: Cross back ties
-  description: Whether you'd like to cross the ties at the back or not
+  description: Whether you'd like to use the cross back tie variation of Bee
+
+bandLength:
+  title: Band Length (Cross back ties)
+  description: Controls the length of the band around your chest for the cross back ties variation of Bee
+
+backDartHeight:
```

Comment:
This part surprised me since I couldn't find these options when configuring Bee. Sorry if I am missing something.

Reply:
Sorry, I added this comment on the wrong line originally and had to delete and re-add it.

Reply:
They are in the advanced section. Due to Bee being made out of strectch fabrics and a lot of it's fit is to do with gathering the default Bella for the most part doesn't affect Bee but these are the few that do so they are added in case someone has settings they want to bring across from Bella.

Reply:
Oh I see it now. And this change makes those options prettier with real names and descriptions. Makes sense. Thanks for explaining.

Reply:
That's alright. Since I had done Holmes yesterday I thought I would update Bee as well. It's sort of a distraction from some of the tougher docs work I am working on. Plus now Holmes is one step closer to full completion which makes me happy :)